packages/location-field/src/stories/Desktop.story.js

```diff
@@ -82,6 +90,14 @@ export const AutoFocusWithMultipleControls = () => (
     />
   </div>
 );
+AutoFocusWithMultipleControls.parameters = {
+  a11y: {
+    config: {
+      // This is a story issue, not a production issue
+      rules: [{ id: "label", enabled: false }]
+    }
+  }
+};
```

Comment:
Extract the object assigned to `.parameters` to a `const`, and reuse.

Reply:
Done in 59cfcfb631e5a0f6c6ef52bc7260faf988bcd5bd